frontend/components/AbsoluteSlider.hpp

```diff
@@ -4,11 +4,18 @@
 
 class AbsoluteSlider : public SliderIgnoreScroll {
 	Q_OBJECT
+	Q_PROPERTY(QColor tickColor READ getTickColor WRITE setTickColor DESIGNABLE true)
 
 public:
 	AbsoluteSlider(QWidget *parent = nullptr);
 	AbsoluteSlider(Qt::Orientation orientation, QWidget *parent = nullptr);
 
+	QColor getTickColor() const;
+	void setTickColor(QColor c);
+
+	bool getDisplayTicks() const;
+	void setDisplayTicks(bool display);
```

Comment:
Prefer these in the same order here and in `frontend/components/AbsoluteSlider.cpp`.